src/services/queries/auth/useSignin.ts

```diff
@@ -0,0 +1,92 @@
+import { useMutation } from "@tanstack/react-query";
+import { useRouter } from "next/router";
+
+import useAlertModal from "@/hooks/useAlertModal";
+import { useAppDispatch } from "@/redux/hooks";
+import { setUser } from "@/redux/slices/userSlice";
+import { adminSignIn, artistSignIn } from "@/services/api/requests/auth/auth.post.api";
+import {
+  IPostAdminSignInRequest,
+  IPostAdminSignInResponse,
+  IPostArtistSignInRequest,
+  IPostArtistSignInResponse,
+} from "@/services/api/types/auth";
+import { MODAL_TYPE } from "@/types/enums/modal.enum";
+import { MEMBER_TYPE, MemberType } from "@/types/enums/user.enum";
+import { setCookie } from "@/utils/cookies";
+import getLatestYearMonthString from "@/utils/getLatestYearMonthString";
+
+const useAdminSignin = () => {
+  const dispatch = useAppDispatch();
+  const router = useRouter();
+  const { showAlertModal } = useAlertModal({
+    type: MODAL_TYPE.ERROR,
+    title: "로그인 실패",
+    message: "아이디와 비밀번호를 확인하세요",
+  });
+  const mutation = useMutation<
+  IPostAdminSignInResponse,
+  unknown,
+  IPostAdminSignInRequest,
+  unknown
+  >(
+    ["admin", "signin"],
```

Comment:
```suggestion
    [MEMBER_TYPE.ADMIN, "signin"],
```
유저 타입은 타입 선언했던 걸로 써주시면 될 것 같습니당